pkg/util/hostname/README.md

```diff
@@ -92,15 +92,15 @@ The results are as follows:
 | none      | false                 | 1         | os            | none          |
 | none      | true                  | 1         | os            | none          |
 | v2 only   | false                 | 1         | os            | none          |
-| v2 only   | true                  | 1         | aws (i-..)    | aws (i-..)    |
+| v2 only   | true                  | 1         | os            | none          |
 | v1+v2     | false                 | 1         | aws (i-..)    | aws (i-..)    |
 | v1+v2     | true                  | 1         | aws (i-..)    | aws (i-..)    |
 | none      | false                 | 2+        | os            | none          |
 | none      | true                  | 2+        | os            | none          |
 | v2 only   | false                 | 2+        | os            | none          |
-| v2 only   | true                  | 2+        | os            | none          |
-| v1+v2     | false                 | 2+        | os            | aws (i-..)    |
-| v1+v2     | true                  | 2+        | os            | aws (i-..)    |
+| v2 only   | true                  | 2+        | aws (i-..)    | aws (i-..)    |
+| v1+v2     | false                 | 2+        | aws (i-..)    | aws (i-..)    |
+| v1+v2     | true                  | 2+        | aws (i-..)    | aws (i-..)    |
```

Comment:
@ian28223 I think these might be flipped. I think the `hops` value here is how many hops the agent has to make in a default setting, not what the hop limit setting is. If that's the case, you may have the changes reversed.